crates/shared/src/transport/http.rs

```diff
@@ -303,4 +309,31 @@ mod tests {
         )
         .is_err());
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn batch_call_success() {
+        let http = create_env_test_transport();
+        let web3 = Web3::new(http);
+        let request = web3.transport().prepare("eth_blockNumber", Vec::default());
+        let request2 = web3.transport().prepare("eth_chainId", Vec::default());
+        let _batch_response = web3
+            .transport()
+            .send_batch([request, request2])
+            .await
+            .unwrap()
+            .into_iter();
```

Comment:
Since we don't actually care about the result `into_iter()` and even the assignment to a local variable can be skipped.

```suggestion
            .unwrap();
```